issuance/message/example/credential-request-message.json

```
@@ -0,0 +1,7 @@
+{
+    "@context": "http://example.org/context",
```

```suggestion
    "@context": ["https://w3id.org/tractusx-trust/v0.8", "http://example.org/context"],
```
`@context` should be an array in all examples and always hold the `tractusx-trust` URI.

---

done